schema/schema.json

```diff
@@ -1987,6 +1987,12 @@
                         "type": {
                             "const": "object",
                             "type": "string"
+                        },
+                        "required": {
+                            "items": {
+                                "type": "string"
+                            },
+                            "type": "array"
```

Comment:
Note that schema.json is generated from schema.ts. Please follow [CONTRIBT](https://github.com/modelcontextprotocol/specification/blob/main/CONTRIBUTING.md)

Reply:
Got it! [This should do it](https://github.com/modelcontextprotocol/specification/pull/87/commits/d110191ba4ad280ab2282b228975a60f01187235) , thanks